javascript/computer_science/hash_map_data_structure.md

```diff
@@ -168,17 +168,17 @@ For example, if we are to find the bucket where the value `"Manon"` will land, t
 
 As we continue to add nodes into our buckets, collisions get more and more likely. Eventually, however, there will be more nodes than there are buckets, which guarantees a collision (check the additional resources section for an explanation of this fact if you're curious).
 
-Remember we don't want collisions. In a perfect world each bucket will either have 0 or 1 node only, so we grow our buckets to have more chance that our nodes will spread and not stack up in the same buckets. To grow our buckets, we create a new buckets list that is double the size of the old buckets list, then we copy all nodes over to the new buckets.
+Remember we don't want collisions. In a perfect world, each bucket will either have 0 or 1 node only, so we grow our hash table to have more chance that our nodes will spread and not stack up in the same buckets. To grow our hash table, we create a new one that is double its size and then copy all existing nodes over to the buckets of this new table, hashing their keys again.
```

Comment:
I see what you're going for and why you think the current wording is a problem, but I think changing to `hash table` is also potentially confusing. Learners might interpret that to mean they need to create an entirely new `HashMap` instance and copy entries over to it somehow.

What if you called it "buckets array" in this section? This should make clear it's the containing array that needs to be changing rather than anything about individual buckets. Let me know what you think.

Reply:
Thank you for the feedback. I'll make the proposed changes. Sorry for the delay!